_includes/compact_hub_cards.html

```diff
@@ -0,0 +1,20 @@
+<div class="compact-hub-index-cards" id="hub-cards">
+  <div class="list compact-cards cards-left">
+  {% assign hub = site.hub | where: "category", include.category | sort: "order" %}
+
+  {% for item in hub %}
+    <div class="col-md compact-hub-card-wrapper" data-item-count="{{ forloop.index }}" data-tags="{{ item.tags | join: ',' }}">
+      <div class="compact-model-card">
+        <a href="{{ site.baseurl }}{{ item.url }}">
+          <div class="compact-card-body">
+            <div class="compact-card-container">
+              <h4 class="compact-item-title col-6">{{ item.title }}</h4>
+              <p class="compact-card-summary card-summary col-6">{{ item.summary }}</p>
+            </div>
+          </div>
+        </a>
+      </div>
+    </div>
+  {% endfor %}
+  </div>
+</div>
```

Comment:
There should be a little more padding between the last card row and the footer. You can look at the resources page to get an idea.